perma_web/perma/tasks.py

```diff
@@ -940,26 +918,23 @@ def _proxy_request(self):
             print "x-robots-tag found, darchiving"
 
         if have_html:
+
             # Get a copy of the page's metadata immediately, without
             # waiting for the page's onload event (which can take a
             # long time, and might even crash the browser)
-            with browser_running(link, browser):
-                print "Retrieving page metadata (first time)"
-                add_thread(thread_list, get_metadata, args=(
-                    page_metadata,
-                    link,
-                    browser
-                ))
```

Comment:
Super nitpick: may we have a print statement here, something like "Retrieving DOM (pre-onload)" or similar, and a post-load equivalent the next time this runs? I like seeing where in the capture process the browser crashes.